charts/drax/configuration/grafana/dashboards/5g-du-metrics.json

```
@@ -4814,8 +4831,8 @@
     ]
   },
   "timezone": "",
-  "title": "DU Metrics",
-  "uid": "a8a69f8a-1dd1-4496-8f3f-33de8988a823",
-  "version": 1,
+  "title": "DU Metrics - MOD",
+  "uid": "a8a69f8a-1dd1-4496-8f3f-33de8988a8231",
```

This should be reverted:

```suggestion
  "title": "DU Metrics",
  "uid": "a8a69f8a-1dd1-4496-8f3f-33de8988a823",
```